src/blurredLocationDisplay.js

```diff
@@ -7,11 +7,35 @@ BlurredLocationDisplay = function BlurredLocationDisplay(options) {
 
   options.Interface = options.Interface || require('./ui/Interface.js');
   options.blurredLocation = options.blurredLocation || {};
+  options.PLpeopleAPI = options.PLpeopleAPI || false ;
+  options.locations = options.locations || [] ;
+  options.source_url = options.source_url || "" ;
   map = options.blurredLocation.map ;
   var InterfaceOptions = options.InterfaceOptions || {};
   InterfaceOptions.blurredLocation = options.blurredLocation;
   var Interface = options.Interface(InterfaceOptions);
 
+  L.Icon.BlackIcon = L.Icon.extend({
```

Comment:
Should we try to document what we use each color for with a comment in the source?